tests/test_middleware.py

```diff
@@ -286,6 +286,32 @@ def test_magic_response():
         if c.name == 'spam':
             assert c.value == 'ham'
 
+    resp_data = {
+        'html': '<html><body>Hello</body></html>',
```

Comment:
I think it would be nice to check that non-ascii response body is correctly decoded